xrpl/models/requests/ledger_entry.py

```diff
@@ -206,17 +239,28 @@ def _get_errors(self: LedgerEntry) -> Dict[str, str]:
             for param in [
                 self.index,
                 self.account_root,
-                self.directory,
-                self.offer,
-                self.ripple_state,
                 self.check,
+                self.deposit_preauth,
+                self.directory,
                 self.escrow,
+                self.offer,
                 self.payment_channel,
-                self.deposit_preauth,
+                self.ripple_state,
                 self.ticket,
+                self.xchain_claim_id,
+                self.xchain_create_account_claim_id,
             ]
             if param is not None
         ]
-        if len(query_params) != 1:
+        if (
+            len(query_params) != 1
+            and self.bridge is None
+            and self.bridge_account is None
+        ):
             errors["LedgerEntry"] = "Must choose exactly one data to query"
+
+        if (self.bridge is not None) != (self.bridge_account is not None):
```

Comment:
`!=` with booleans is a bit hard to read. Can you split up the check into both None or both not None? 

(As an aside, since both flip flops are part of the !=, if you wanted to simplify it without removing the `!=` you could just get rid of the `not` on both sides)